keyboards/keebio/nyquist/keymaps/georgepetri/readme.md

```diff
@@ -0,0 +1,52 @@
+# George Petri's Quefrency 65 layout
+
+```
+make keebio/quefrency:georgepetri
```

Comment:
You should specify a revision here, because the default set at the keyboard level could change in the future. I'm assuming you have a rev2.

```suggestion
make keebio/nyquist/rev2:georgepetri
```